src/infrastructure/servers/taltres.md

```diff
@@ -0,0 +1,41 @@
+# TalTres
+
+_updated: September 15, 2024_
+
+TalTres is COSI's primary DNS server, replacing TalDos (now Caterpillar).
+It also runs our point-to-site VPN.
+
+*Third time's the charm.*
+
+| | |
+| :--- | :--- |
+| Location | [Server Room - Network 1](../racks.md#network-1) |
+| IP Addresses | 128.153.145.3, 128.153.145.53 |
+| Deployed | true |
+
+## Hardware
+
+| | |
+| :--- | :--- |
+| CPU | Intel Xeon E5-2640v3 (x2)
+| RAM | 64 GB
+| STORAGE | 2.4 TB
+| CONNECTIVITY | 1 Gbps
+
+## Operating System
+
+| | |
+| :--- | :--- |
+| OS | GNU/Linux
+| Distro | Debian 12 Bookworm
+| Last updated | Sep 2024
+| End of life | unknown
+| Enrolled in COSI auth | false
```

Comment:
Does this line need to be included since we don't currently have a central auth service?

Reply:
I would be inclined to keep it there if that's something we want to explore in the future. I'm a lot more confident I can set something like that up and document it now than I was a few months ago, so we could take a look at that soon